test/bin/scenario.sh

```diff
@@ -692,31 +694,13 @@ current_version() {
         error "Run ${SCRIPTDIR}/create_local_repo.sh before running this scenario."
         return 1
     fi
-    "${SCRIPTDIR}/get_latest_rpm_version.sh"
-}
-
-# Function to report only the minor portion of the current version,
-# e.g. from "4.13.5" reports "13"
-current_minor_version() {
-    current_version | cut -f2 -d.
-}
-
-# Function to report the *previous* minor version. If the current
-# version is "4.13.5", reports "12".
-previous_minor_version() {
-    echo $(( $(current_minor_version) - 1 ))
-}
 
-# Function to report the minor version for 2 releases back. If the current
-# version is "4.16.0", reports "14".
-yminus2_minor_version() {
-    echo $(( $(current_minor_version) - 2 ))
-}
-
-# Function to report the *next* minor version. If the current
-# version is "4.14.5", reports "15".
-next_minor_version() {
-    echo $(( $(current_minor_version) + 1 ))
+    local -r release_info_rpm=$(find "${LOCAL_REPO}" -name 'microshift-release-info-*.rpm' | sort | tail -n 1)
```

Comment:
It's not strictly part of the PR, but I find this function name confusing, especially with the comment. It sounds like it gets version of current release's released RPMs, not version of locally built RPMs from source.

Reply:
It's only used in one place. Let me rename it.